uploader/wiki_uploader.py

```diff
@@ -31,8 +31,9 @@ def upload_file(self, file_name, file_stream, description=""):
             file=file_stream,
             filename=file_name,
             description=description,
+            date_created = datecreated,
             ignore=True,
-            comment=get_initial_page_text(datecreated, description),
+            comment=get_initial_page_text(date_created, description),
```

Comment:
it is still undefined. As you can see, when you are in this function, you do not have access to `date_created`

Reply:
I would suggest you do a default date_created = current date time when it is not provided.

Reply:
Okay i will do that